devops/infrastructure/bash-config.sample.cfg

```diff
@@ -5,7 +5,8 @@ REGION=us-east-1
 CLI_PROFILE=joystream-user
 KEY_PATH="/Users/joystream/Joystream/joystream-key.pem"
 AWS_KEY_PAIR_NAME="joystream-key"
-EC2_INSTANCE_TYPE=t2.micro
+NODE_EC2_INSTANCE_TYPE=t2.micro
+BUILD_EC2_INSTANCE_TYPE=t2.xlarge
```

Comment:
We should have a variable for each "role" so in addition to BUILD_EC2_INSTANCE_TYPE:
- VALIDATOR_EC2_INSTANCE_TYPE
- RPC_EC2_INSTANCE_TYPE

and if a value is not set, the colud formation template can fall back to the default.

Reply:
Added option to set RPC, Validator and Build instance types